### ISSUE TITLE
False error, Empty constructor already exists

### ISSUE THREAD

**Issue by rajinder-yadav** [sohibegit/vscode-java-code-generator#33]
Please see screen shot, I am getting this error and unable to create a empty ctor (bottom right).

![snap-36](https://user-images.githubusercontent.com/439226/104079126-78c2d480-51ef-11eb-84fc-3cb54061bc02.png)

Also when I select all the field I get the same error.

![snap-37](https://user-images.githubusercontent.com/439226/104079153-909a5880-51ef-11eb-875e-1d83e29ee746.png)

After I close the GUI and reopen it, I am able to create multiple empty ctor.

![snap-38](https://user-images.githubusercontent.com/439226/104079027-f76b4200-51ee-11eb-9fb4-6916eca84fb4.png)

**Comment by rajinder-yadav** [comment]
I closed the GUI and re-opened it and I have all the fields selected, when I click on the Constructor using field button I get an empty ctor?

```java
  public Person() {
  }
```

**Comment by sohibegit** [comment]
7.0.1 should fix your problems please reopen if not.

**Comment by rajinder-yadav** [comment]
Hello bug still exists in 7.0.1

I generated a empty ctor, then I selected all the fields and tried to generate the ctor and get a error saying "Empty ctro already exists". See pic.

1. Also closing the GUI and re-opening doesn't correct the error.
2. If I delete the generated empty ctor, I am unable to generate a ctor with all fields, I just generates a empty ctor.

![snap-39](https://user-images.githubusercontent.com/439226/104111480-1da4e680-52b0-11eb-83fb-00009c8b4a4b.png)
